docs/pages/kubernetes-access/guides/standalone-teleport.mdx

```diff
@@ -36,73 +44,192 @@ $ kubectl config get-contexts
 Use this command to switch to the cluster assigned to `CONTEXT_NAME`:
 
 ```code
+# e.g., my-context
+$ CONTEXT_NAME=<Var name="context-name" />
 $ kubectl config use-context ${CONTEXT_NAME?}
 ```
 
-Next, use [get-kubeconfig.sh](https://github.com/gravitational/teleport/blob/master/examples/k8s-auth/get-kubeconfig.sh) to create a `kubeconfig` for Teleport to use.
+### Run the script
 
-You can connect multiple Kubernetes clusters to Teleport from one `kubeconfig` if it contains multiple entries. Use
-[merge-kubeconfigs.sh](https://github.com/gravitational/teleport/blob/master/examples/k8s-auth/merge-kubeconfigs.sh) to combine multiple `kubeconfigs` generated by `get-kubeconfig.sh`.
+On your workstation, download Teleport's `get-kubeconfig.sh` script, which you
+will use to generate the `kubeconfig` file:
 
-## Step 2/4. Add your kubeconfig to Teleport
+```code
+$ curl -OL \
+https://raw.githubusercontent.com/gravitational/teleport/master/examples/k8s-auth/get-kubeconfig.sh
```

Comment:
Could we do some magic here to insert the branch name related to the current release of the docs the user is viewing ? This could get confusing if in a future Teleport version, we move this script, which would then break older docs versions.